### PR TITLE
Fixed question description and added util for icons

### PR DESCRIPTION
**Pull request by Mav-Ivan**
image

### CODE REVIEW

**Review by MariaAfonina — APPROVED** [review]

File: src/utils/check-icons.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import CheckCircleOutlineIcon from '@mui/icons-material/CheckCircleOutline'
+import DriveFileRenameOutlineOutlinedIcon from '@mui/icons-material/DriveFileRenameOutlineOutlined'
+import FormatListBulletedOutlinedIcon from '@mui/icons-material/FormatListBulletedOutlined'
+
+export const CheckIcons = (type: string) => {
+  return (
+    (type === 'oneAnswer' && <CheckCircleOutlineIcon />) ||
+    (type === 'openAnswer' && <DriveFileRenameOutlineOutlinedIcon />) ||
+    (type === 'multipleChoice' && <FormatListBulletedOutlinedIcon />)
```
Comment (TSlashDreamy) [inline src/utils/check-icons.tsx:9]: maybe we need to put this string into some sort of `constants` file, or is that redundant?

**Review by TSlashDreamy — APPROVED** [review]

File: src/utils/check-icons.tsx (hunk 1 above)
Comment (Mav-Ivan) [reply src/utils/check-icons.tsx]: yeah , that`s good idea

File: src/constants/index.ts [hunk 2]
```diff
@@ -9,6 +9,12 @@ export const admin = 'admin'
 export const login = 'login'
 export const signup = 'signup'
 
+export const types = {
+  oneAnswer: 'oneAnswer',
+  openAnswer: 'openAnswer',
+  multipleChoice: 'multipleChoice'
+}
```
Comment (dmtrth25) [inline src/constants/index.ts:16]: it looks like enum can use instead

File: src/utils/check-icons.tsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import CheckCircleOutlineIcon from '@mui/icons-material/CheckCircleOutline'
+import DriveFileRenameOutlineOutlinedIcon from '@mui/icons-material/DriveFileRenameOutlineOutlined'
+import FormatListBulletedOutlinedIcon from '@mui/icons-material/FormatListBulletedOutlined'
```
Comment (dmtrth25) [inline src/utils/check-icons.tsx:3]: can destructure imports from '@mui/icons-material'

File: src/utils/check-icons.tsx [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+import { types } from '~/constants'
+
+export const CheckIcons = (type: string) => {
+  return (
+    (type === types.oneAnswer && <CheckCircleOutlineIcon />) ||
+    (type === types.openAnswer && <DriveFileRenameOutlineOutlinedIcon />) ||
+    (type === types.multipleChoice && <FormatListBulletedOutlinedIcon />)
+  )
```
Comment (dmtrth25) [inline src/utils/check-icons.tsx:12]: I suppose use switch case here more preferable
`switch (type) {} ` etc.

File: src/utils/check-icons.tsx (hunk 3 above)
Comment (TSlashDreamy) [reply src/utils/check-icons.tsx:3]: > can destructure imports from '@mui/icons-material'

i think we can leave it like this, we need to practice treeshaking, so this implementation seems fine

File: src/constants/index.ts (hunk 2 above)
Comment (Mav-Ivan) [reply src/constants/index.ts]: done

File: src/containers/my-resources/questions-container/QuestionsContainer.constants.tsx [hunk 5]
```diff
@@ -13,21 +12,20 @@ import {
...
 import { getFormattedDate } from '~/utils/helper-functions'
+import { CheckIcons } from '~/utils/check-icons'
 import { styles } from '~/containers/my-resources/questions-container/QuestionsContainer.styles'
 
 export const columns: TableColumn<Question>[] = [
   {
     label: 'myResourcesPage.questions.title',
     field: 'title',
     calculatedCellValue: (item: Question) => {
+      const icon = CheckIcons(item.type)
```
Comment (Tolik170) [inline src/containers/my-resources/questions-container/QuestionsContainer.constants.tsx:23]: maybe just pass CheckIcons(item.type) below into icon and remove {} brackets to return instanty

**Review by Tolik170 — APPROVED** [review]

File: src/containers/my-resources/questions-container/QuestionsContainer.constants.tsx (hunk 5 above)
Comment (Mav-Ivan) [reply src/containers/my-resources/questions-container/QuestionsContainer.constants.tsx]: done

**Review by dmtrth25 — APPROVED** [review]

**Review by maarekua — APPROVED** [review]

**Pull request merged**